src/cargo/util/command_prelude.rs

```diff
@@ -331,7 +331,16 @@ pub trait CommandExt: Sized {
     }
 
     fn arg_quiet(self) -> Self {
+        let unsupported_silent_arg = || {
```

Comment:
Why do we need a closure here?

Reply:
I want to add the `quiet` flag first, but I guess it doesn't matter.